supplementary/devices/cedric.rst

```diff
@@ -3,8 +3,11 @@ Cedric - Motorola Moto G5
 =========================
 
 The Moto G5 is a mid-range phone, with a few variants.
-
-The XT1676 variant is special in that it has 3 GB of RAM.
```

Comment:
Should we keep the detail that 1676 has 3gb somewhere?

Reply:
I have an XT1676 with 2GB.